proto/scheduler.proto

```diff
@@ -95,6 +95,14 @@ message TaskSize {
 
   // The task's estimated disk space requirement (beyond task inputs).
   int64 estimated_free_disk_bytes = 3;
+
+  // Whether the memory estimate comes from an explicit request parameter as
+  // opposed to an automatic estimate.
+  bool explicit_memory = 4;
```

Comment:
Thoughts on using separate `estimated_` and `requested_` fields here instead of the booleans?

I think either way is fine, just curious about the tradeoff.

Reply:
I thought that recording the raw resource requests as integer fields might get confusing, because we apply caps to certain resources, and IMO it'd be unclear whether these fields are pre- or post- cap. Also, having explicit values would open up more opportunity to use those requested values in places that we shouldn't be.